docs/community/developer_guide.mdx

```diff
@@ -66,70 +66,70 @@ Windows:
 make build-windows
 ```
 
-## Cleaning workspace
+## Cleaning Workspace
 
 To remove all files not manged by git, run `make clean` (be careful!)
 
-## Managing dependencies
+## Managing Dependencies
 
 [Using Go Modules](https://blog.golang.org/using-go-modules) to manage dependencies.
 
 To update or add new dependencies, run `go get <package name>`.
 
-## Release checklist
+## Release Checklist
 
 :::info
 
 This section needs a lot of love and automation 🙂
 
 :::
 
```

Comment:
These should automatically number when they start with `1.`. Is that not working?

Reply:
It is, I thought that was done by accident in the markdown though. I'll revert these changes in the next commit.